constants/defaults.py

```diff
@@ -80,6 +80,15 @@
 # Enums
 HTK_ENUM_SYMBOLIC_NAME_OVERRIDES = {}
 
+##
+# JS Routes
+HTK_JS_ROUTES_ENABLED = False
```

Comment:
Why does this need to be a setting? It will be fully controlled code. If we don't want to print out any routes, don't call the function in the first place.

Unless you were thinking of always adding this to certain base templates? In which case, I would still argue that we should just add it to templates when we explicitly want to do it.

Reply:
Yes you are correct. It does not makes sense. Removed.